VideoLocker/src/main/java/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -292,7 +293,8 @@ private void createAccount() {
             PrefManager pref = new PrefManager(this, PrefManager.Pref.LOGIN);
             String access_token = pref.getString(PrefManager.Key.AUTH_TOKEN_SOCIAL);
             String backstore = pref.getString(PrefManager.Key.AUTH_TOKEN_BACKEND);
-            if ( access_token != null && access_token.length() > 0 ) {
+            boolean fromSocialNet = TextUtils.isEmpty( access_token );
```

Comment:
I guess the code is already like this, but maybe there's a better way to figure out if we're doing social registration? It kind of error prone, since it's all going through a dictionary. It's not clear to me we need to save their social access token anyway? Typically the google or facebook library will save that for us